com.byteseb.Duel.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.byteseb.Duel",
```

Comment:
`byteseb.com` doesn't seem to exist, so the app-id doesn't meet the requirements.

Reply:
Until you fix this, this PR can't move forward.